doc/index.rst

```diff
@@ -110,6 +129,9 @@ Additional Options for :rst:dir:`.. autoprogram::`
 
 .. versionadded:: 0.1.3
 
+``:groups:``
+    Render argument groups as subsections.
```

Comment:
Could you add `.. versionadded:: 0.15` here?

Reply:
Sure! Should I then move the above `versionadded:: 0.1.3` to each existing attribute, to avoid confusion?

Reply:
That would be better indeed!